README.md

```diff
@@ -25,24 +31,15 @@ Ensure your machine satisfies the requirements:
 - [Docker Engine](https://docs.docker.com/engine/install/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 
-## Memeooorr Agent: Functionality Overview
-
-**Supported Chains:**
-The Memeooorr agent currently operates on the following chains:
-- Base
-- Celo
-
-See [here](https://github.com/dvilelaf/meme-ooorr?tab=readme-ov-file#user-flow) for what it does.
-
-Some thing you will need:
+## Setup Requirements
 
-- For the initial setup you will need to fund certain addresses with some funds when requested.
+- For the initial setup you will need to fund certain addresses with the following funds when requested: 0.05 ETH (Base chain) or x CELO (Celo chain). These quantities are based on the gas prices seen on the 1st half of Sept 2024 and may need to be revised. Additionally some quantity of OLAS bridged to Base or Celo if you want to stake.

```

Comment:
What are funding requirements for CELO?

Reply:
HAvent tested on Celo. @0xArdi have you?

Reply:
I'll merge so I can tag 0.1.0, then we can fix on another PR